### PR TITLE
Fix test

### PR DESCRIPTION
**Pull request by btzr-io**
## Fixes
- Fix components test
- Fix button style
- Update styles
- Add native tooltips

### CODE REVIEW

File: src/components/button/index.js [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
+import style from './style.styl';
 
 class Button extends Component {
+
+    static defaultProps = {
+        type: 'normal',
+        icon: false,
+        loading: false,
+        apply: () => {}
+    }
```
Comment (xaiki) [inline src/components/button/index.js]: we should probably add `prop-types` https://facebook.github.io/react/docs/typechecking-with-proptypes.html

File: src/components/switch/index.js [hunk 2]
```diff
@@ -3,16 +3,22 @@ import { translate } from 'react-i18next';
...
+    }
+
     constructor (props) {
         super()
         this.state = {
-            selected: props.selected || false
+            selected: props.selected
         }
-        this.apply = props.apply || function () {}
+        this.apply = props.apply
```
Comment (xaiki) [inline src/components/switch/index.js]: i guess it's probably a good idea to: `this.apply = props.apply.bind(this)`

File: src/components/button/index.js [hunk 3]
```diff
@@ -1,29 +1,37 @@
...
+
     constructor (props) {
         super()
-        this.apply = props.apply || function () {}
+        this.apply = props.apply
```
Comment (xaiki) [inline src/components/button/index.js]: i guess it's probably a good idea to: `this.apply = props.apply.bind(this)`

File: src/components/button/index.js [hunk 4]
```diff
@@ -58,22 +66,9 @@ class ActionButton extends Component {
...
-                <span>{props.t(props.title)}</span>
-            </a>
-        )
-    }
-}
-
 let Buttons = {
     Button: translate(['button'], {wait: true, withRef: true})(Button),
-    ActionButton: translate(['action-button'], {wait: true, withRef: true})(ActionButton),
-    GoBackButton: translate(['go-back-button'], {wait: true, withRef: true})(GoBackButton)
```
Comment (xaiki) [inline src/components/button/index.js:76]: why kill GoBackButton ?

File: src/components/button/style.styl [hunk 5]
```diff
@@ -34,38 +34,47 @@
...
+        border: 2px solid var(--Button-bg)
+
+        &:hover
+        &:active
+            background: var(--Button-bg)
+            color: var(--Button-color)
 
 // Go-back button
 
 .go-back
```
Comment (xaiki) [inline src/components/button/style.styl]: GoBackButton killed but CSS still present

File: src/components/dropdown/index.js [hunk 6]
```diff
@@ -34,17 +34,27 @@ const DropdownToggle = (props) => (
...
+    }
+
     constructor (props) {
         super()
         this.state = {
             open: false,
             selected: props.selected || Object.keys(props.options)[0]
         }
-        this.apply = props.apply || function () {}
+        this.apply = props.apply
```
Comment (xaiki) [inline src/components/dropdown/index.js]: i guess it's probably a good idea to: `this.apply = props.apply.bind(this)`

**Review by xaiki — CHANGES_REQUESTED** [review]
looks good, a few nitpicks

File: src/components/button/index.js (hunk 4 above)
Comment (btzr-io) [reply src/components/button/index.js:76]: here -> [src/components/navbar/index.js#L6](https://github.com/buttercomponents/butter-base-components/blob/master/src/components/navbar/index.js#L6)

Comment (xaiki) [reply src/components/button/index.js:76]: ok i guess it makes sense

File: src/components/button/index.js [hunk 7]
```diff
@@ -1,29 +1,45 @@
...
 class Button extends Component {
+
+    static defaultProps = {
+        type: 'normal',
+        loading: false,
+        apply: () => {}
+    }
+
+    static propTypes = {
+        type: PropTypes.string.isRequired,
```
Comment (xaiki) [inline src/components/button/index.js:16]: i thinks `isRequired` means you **need** to pass it everytime, here we have it in default props

**Review by xaiki — APPROVED** [review]
left a small comment if you think it's ok you can merge

**Comment by xaiki** [comment]
```
The defaultProps will be used to ensure that this.props.name will have a value if it was
not specified by the parent component. The propTypes typechecking happens after
defaultProps are resolved, so typechecking will also apply to the defaultProps.
```

File: src/components/button/index.js (hunk 7 above)
Comment (btzr-io) [reply src/components/button/index.js:16]: ok, anyway you merged this 🙃

Comment (xaiki) [reply src/components/button/index.js:16]: yeah, i put the explanation in the thread =P

**Pull request merged**